notebooks/official/experiments/comparing_pipeline_runs.ipynb

```diff
@@ -2,7 +2,8 @@
   "cells": [
```

Comment:
<h3>remove heading: Define your workflow using Kubeflow Pipelines DSL package</h3><p> </p><p>Combine with above text cell.</p><p> </p><p>Add explanation of the step.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/759/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>